### DISCUSSION TITLE
New Install Error

### DISCUSSION THREAD

**Discussion by currong** [opencart/opencart#12335] [Q&A]
Unknown: mysqli::real_escape_string(): Passing null to parameter #1 ($string) of type string is deprecated in /home/bandmecr/public_html/OpenCart/system/library/db/mysqli.php on line 53

I am getting this message at the top of home page on new install of v3.0.3.8

Any thoughts on how to fix this

**Comment by mhcwebdesign** [comment]
Use the OpenCart forum for support.

Anyway, you are probably using PHP 8 and OpenCart 3.0.3.8, which won't work. Either use PHP 7.4, or use the 3.0.x.x_Maintenance release which supports PHP 8.

**Reply by currong** [reply]
Problem fixed.  Thank you for your reply.  I apologise for using the wrong forum for support.

**Reply by punitkumar** [reply]
Is 3.0.x.x_Maintenance available to download through Opencart? 
Is it safe to use it? it is updated 2 weeks ago.

https://github.com/opencart/opencart-3

**Reply by ADDCreative** [reply]
You will find the 3.0.x.x_Maintenance version at https://github.com/opencart/opencart/tree/3.0.x.x_Maintenance.
